### PR TITLE
Carolyn H - Editing README.md

### PR DESCRIPTION
**Pull request by carolynmh**
I used basic R Markdown tips found using the GitHub guide. I initially had a few issues with figuring out the image sourcing process when pulling the image I wanted from the Internet, briefly considered using an image from my own computer, but eventually went back into Stack Overflow to find a solid description of how to pull an image file from the Internet. After some troubleshooting, I was able to finally get the result I wanted. I also ran into an issue with a link I had used, but this was an easier mistake to solve as it was due to a typo. I think once I got used to the formatting in R Markdown, the process became a lot quicker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+## hw01 - editing README.md
+
+-----
+
+# My name is Carolyn Hammond.
+
+![](https://66.media.tumblr.com/e1d62157f9298082303d249d1872f3a5/tumblr_mkqa55nS1n1snqyxgo1_1280.png)
```
Comment (Jade-Zhou) [inline README.md:7]: A nice one to add the image from the internet!

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
-# hw01
```
Comment (Jade-Zhou) [inline README.md:1]: Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | X |   |  
Coding strategy | X |   |  
Presentation | X |   |  
Achievement, creativity | X |   |  
Ease of access | X

**Comment by Mirrorlkj** [comment]
valuation
----------------------------------------------------
  | Topic                       | Excellent | Satisfactory | Needs Work |
  |-----------------------------|-----------|--------------|------------|
  | **Coding style**            |     X      |              |            |
  | **Coding strategy**         |     X      |              |            |
  | **Presentation**            |      X     |              |            |
  | **Achievement, creativity** |      X     |              |            |
  | **Ease of access**          |      X     |              |            |

 Remarks:

* Nice work you've done! headings, lists, pics and all.
* I learned from you how to add a pic from the internet, I did it differently by firstly adding it into the private repository.
* To be more reader-friendly, it would be better if you only use bold in headings.
* Appreciate what you are doing and trying to do!

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+
+# My name is Carolyn Hammond.
+
+![](https://66.media.tumblr.com/e1d62157f9298082303d249d1872f3a5/tumblr_mkqa55nS1n1snqyxgo1_1280.png)
+
+-----
+
+## School-Related Endeavors 
+
+### I am a **third year** at the University of Chicago double-majoring in **Public Policy** and **Linguistics**. This quarter in particular, I'm taking two *Chicago Studies* courses:
```
Comment (Mirrorlkj) [inline README.md:13]: Just for your future reference,  try not to make the body into headings to avoid confusion

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |             |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |        x      |            |
| **Achievement, creativity** |           |        x      |            |
| **Ease of access**          |     x      |              |            |

Remarks:
1. You have experimented with more than four aspects of markdown language. Great work!
1. Like your peer pointed out, it is a bit confusing to make your body part of the heading. Try to avoid that in the future. 
1. Your commit history is clear and helpful. Keep doing that!

**Pull request closed without merging**